### ISSUE TITLE
The docs say that decreasing num_epoch makes the traning more stable

### ISSUE THREAD

**Issue by orenong** [Unity-Technologies/ml-agents#4527]
**Number of passes to make through the experience buffer when performing gradient descent optimization.The larger the batch_size, the larger it is acceptable to make this. Decreasing this will ensure more stable updates, at the cost of slower learning.**

this is from the docs: https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Training-Configuration-File.md

but I do not understand why, or is it a mistake? why would more passes make something faster but less stable? I don't understand how it works

**Comment by hvpeteet** [comment]
Hi, thanks for reaching out. Since this is more of a question rather than a feature request or bug would you mind moving it to the [forums](https://forum.unity.com/forums/ml-agents.453/)? This way if someone else has the same question it will continue to show up and be searchable in the future. 

Thanks so much for your interest

**Comment by orenong** [comment]
Thank you, I posted it on the forum.

**Comment by hvpeteet** [comment]
Thanks so much.
